webplugin/js/app/mck-app.js

```diff
@@ -146,30 +146,33 @@ function ApplozicSidebox() {
         }
     };
     function addScriptInsideHtml() {
-        var script =  
-        "function showAfterLoad(){"+
-            "var mckSidebox = document.getElementById(\"mck-sidebox\");"+
-            "mckSidebox.style.visibility='visible';"+
-            "var mckLocBox = document.getElementById(\"mck-loc-box\");"+
-            "mckLocBox.style.visibility='visible';"+
-            "var mckGmSearchBox = document.getElementById(\"mck-gm-search-box\");"+
-            "mckGmSearchBox.style.visibility='visible';"+
-        "};"+
-        "if (navigator.userAgent.indexOf('MSIE') !== -1 || navigator.appVersion.indexOf('Trident/') > 0) {"+
-            "showAfterLoad();"+
-        "} else {"+
-            "var isScriptV2 = !!parent.document.getElementById('kommunicate-widget-iframe');"+
-            "if (isScriptV2) {"+
-                "window.parent.document.addEventListener('kmInitilized', function () {"+
-                    "showAfterLoad();"+
-                "}, false);"+
-            "} else {"+
-                "window.addEventListener('kmInitilized', function () {"+
-                    "showAfterLoad();"+
-                "}, false);"+
-            "}"+
-        "}";
+        var scriptData = function detectBrowserAndMakeUiVisible() {
+            function showAfterLoad() {
+                var mckSidebox = document.getElementById("mck-sidebox");
+                mckSidebox.style.visibility = 'visible';
+                var mckLocBox = document.getElementById("mck-loc-box");
+                mckLocBox.style.visibility = 'visible';
+                var mckGmSearchBox = document.getElementById("mck-gm-search-box");
+                mckGmSearchBox.style.visibility = 'visible';
+            };
+            if (navigator.userAgent.indexOf('MSIE') !== -1 ||
+                navigator.appVersion.indexOf('Trident/') > 0) {
+                showAfterLoad;
+            } else {
+                var isScriptV2 = !!parent.document.getElementById('kommunicate-widget-iframe');
+                if (isScriptV2) {
+                    window.parent.document.addEventListener('kmInitilized', function () {
+                        showAfterLoad();
+                    }, false);
+                } else {
+                    window.addEventListener('kmInitilized', function () {
+                        showAfterLoad();
+                    }, false);
+                }
+            };
+        };
 
+        var script = String(scriptData) + "detectBrowserAndMakeUiVisible();"
```

Comment:
Can't create an immediately invoked function because it will get executed when assigned to a variable or passed in String constructor it will get executed. That's why added the function call separately.